inc/Engine/License/ServiceProvider.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace WP_Rocket\Engine\License;
+
+use WP_Rocket\Engine\Container\ServiceProvider\AbstractServiceProvider;
+use WP_Rocket\Engine\License\API\PricingClient;
+
+/**
+ * Service Provider for the License module
+ *
+ * @since 3.7.3
+ */
+class ServiceProvider extends AbstractServiceProvider {
+	/**
+	 * Aliases the service provider provides
+	 *
+	 * @var array
+	 */
+	protected $provides = [
+		'pricing_client',
+	];
+
+	/**
+	 * Registers items with the container
+	 *
+	 * @return void
+	 */
+	public function register() {
+		$views = __DIR__ . '/views';
```

Comment:
`$views` isn't used.

Reply:
Right, it's for later, but I removed it from this PR for now